Source/quests.cpp

```diff
@@ -658,12 +658,6 @@ void ResyncMPQuests()
 		nakrulQuest._qactive = QUEST_ACTIVE;
 		NetSendCmdQuest(true, nakrulQuest);
 	}
-
-	auto &cowQuest = Quests[Q_JERSEY];
-	if (cowQuest._qactive == QUEST_INIT && currlevel == cowQuest._qlevel - 1) {
-		cowQuest._qactive = QUEST_ACTIVE;

```

Comment:
Potentially this was meant to be set to QUEST_HIVE_ACTIVE to indicate the bomb can be spawned? The intent might've been to skip the tease states and just jump straight into the quest.

Reply:
I think it's more likely that they simply copied this section for all the Hellfire quests and never really tested multiplayer, since all Hellfire quests are horribly broken in multiplayer and no other quest is fast forwarded by this section of the code.

Reply:
On a side note the Jersey quest also uses _qactive to track the quest state rater then using _qvar1/2 meaning that it is the only quest that has _qactive values other then inactive/active/completed.